examples/ping/unfinished/icmp.rflx

```diff
@@ -0,0 +1 @@
+../specs/icmp.rflx
```

Comment:
What is that for? Do we want to keep the `unfinished` directory at all? It is part of the demo, but not part of the example. I don't think we want to document the complete demo here.

Reply:
It's needed to support checking the IPv4 spec extended by ICMP (AFAIK we don't have search paths in RecordFlux).

On the one hand, I agree that the files in `unfinished` are not very helpful alone. On the other hand, it would be good to have them somewhere.

Reply:
The "unfinished" files should be stored together with the "coreography". I don't think this repository is the right place for this.

Reply:
OK, I put them there and removed them from the repo in 80ddda8.